package.json

```diff
@@ -45,6 +45,7 @@
     "beachball": "^2.31.0",
     "cross-env": "^7.0.3",
     "patch-package": "^6.4.7",
-    "style-dictionary": "3.1.1"
+    "style-dictionary": "3.1.1",
+    "typescript": "~4.6.4"
```

Comment:
Something in the tests pulls TypeScript from the root node modules, so the tests fail without this despite declaring there own version of TypeScript. I will file an issue to track tech debt, because this can likely be removed once the fast-foundation issue is resolved.